### ISSUE TITLE
Train-test splitting

### ISSUE THREAD

**Issue by project-engage** [project-engage/engage#7]
Split data in training and test set at the date level - i.e. use  the last week of the data period  for testing and the data since December 2019 as training set
